code/style.css

```diff
@@ -1,58 +1,226 @@
+}
+
+.title:hover {
+  color:#EBEBF2;
+  cursor: help;
+}
+
+.title:hover:after {
+  width: 100%;
+  background: #EBEBF2F2;
+}
+
+/****************** CARDS STYLING *******************/
+.card-title {
   margin: 1px;
   font-weight: 300;
-  background-color: #fff;
+  font-size: 25px;
+  background-color:#D1D0D9;
+  text-transform: uppercase;
+}
+
+p {
+  margin: 1px;
+  font-weight: 200;
+  font-size: 25px;
+  text-transform: capitalize;
 }
 
 .cards {
-  margin: 50px auto;
-  width: 90%;
-  height: 600px;
-  background: #ccc;
+  padding: 30px;
   display: flex;
   justify-content: space-around;
   align-items: center;
+  flex-wrap: wrap;
 }
 
 .killer-deck {
   width: 250px;
   height: 400px;
-  background: #000;
+  background:#EBEBF2;
   text-align: center;
   padding-top: 30px;
-  border-radius: 16px;
+  border-radius: 10px;
+  cursor: pointer;
+  transition: transform 2s;
+}
+
+.killer-deck:hover {
+  transform: scale(1.1); 
 }
 
 .weapon-deck {
   width: 250px;
   height: 400px;
-  background: rgb(10, 53, 10);
+  background: #EBEBF2;
   text-align: center;
   padding-top: 30px;
-  border-radius: 16px;
+  border-radius: 10px;
+  cursor: pointer;
+  transition: transform 2s;
+}
+
+.weapon-deck:hover {
+  transform: scale(1.1); 
 }
 
 .room-deck {
   width: 250px;
   height: 400px;
-  background: midnightblue;
-
+  background: #EBEBF2;
   text-align: center;
   padding-top: 30px;
-  border-radius: 16px;
+  border-radius: 10px;
+  cursor: pointer;
+  transition: transform 2s;
+}
+
+.room-deck:hover {
+  transform: scale(1.1); 
+}
+
+.revealed-content {
+  text-align: center;
+  font-size: 25px;
 }
 
-button {
+.button-effect {
   padding: 8px 16px;
-  background: crimson;
-  border: none;
-  font-size: 20px;
-  border-radius: 8px;
+  border: 1px solid white;
+  margin-top: 10px;
+  font-size: 30px;
+  font-weight: 700;
+  font-family: 'Amatic SC', cursive;
   display: block;
   margin: 0 auto;
+  transition: transform 2s;
+  cursor: pointer;
+  background: black;
+  color: #EBEBF2;
+  text-transform: uppercase;
+}
+
+button:focus {
+  outline: 0;
+}
+
+.button-effect:hover {
+  transform: scale(1.2); 
+  color: lightgrey;
+}
+
+.startover-button {
+  padding: 8px 16px;
+  border: none;
+  font-size: 25px;
+  font-family: 'Amatic SC', cursive;
+  display: inline-block;
+  margin: 0 auto;
+  transition: transform 2s;
+  cursor: pointer;
+  background: black;
+  color: white;
+  text-transform: uppercase;
+  text-decoration: underline white;
+}
+
+.startover-button:hover {
+  transform: scale(1.2); 
+  color: #EBEBF2;
+}
+
+.revealed-mystery {
+  margin-bottom: 20px;
+}
+
+/************ MEDIA QUERIES **************/
+
+@media (max-width: 860px) {
+body {
+  background: black;
+}
+#video-background {
+  display: none;
+}
+.content {
+  position: inherit;
+}
+
+.killer-deck {
+  margin-bottom: 20px;
+}
+
+.weapon-deck {
+  margin-bottom: 20px;
+}
+
+h1 {
+  display: none;
+}
+
+.revealed-mystery {
+  font-size: 18px;
+}
```

Comment:
`.revealed-mystery` is defined twice in the CSS.  Maybe once is enough. 
Also, is there a way to prevent the cards from "hopping up" when you click reveal?  Maybe you can do that here somehow?